code/modules/antagonists/eldritch_cult/eldritch_knowledge.dm

```diff
@@ -0,0 +1,248 @@
+  *
+  * Gives addtional effects to sickly blade weapon
+  */
+/datum/eldritch_knowledge/proc/on_eldritch_blade(atom/target, mob/user, proximity_flag, click_parameters)
+	return
+
+//////////////
+///Subtypes///
+//////////////
+
+/datum/eldritch_knowledge/spell
+	var/obj/effect/proc_holder/spell/spell_to_add
+
+/datum/eldritch_knowledge/spell/on_gain(mob/user)
+	var/obj/effect/proc_holder/S = new spell_to_add
+	user.mind.AddSpell(S)
+	return ..()
+
+/datum/eldritch_knowledge/spell/on_lose(mob/user)
+	user.mind.RemoveSpell(spell_to_add)
+	return ..()
+
+/datum/eldritch_knowledge/curse
+	var/timer = 5 MINUTES
+	var/list/fingerprints = list()
+
+/datum/eldritch_knowledge/curse/recipe_snowflake_check(list/atoms, loc)
+	fingerprints = list()
+	for(var/X in atoms)
+		var/atom/A = X
+		fingerprints |= A.return_fingerprints()
+	listclearnulls(fingerprints)
+	if(fingerprints.len == 0)
+		return FALSE
+	return TRUE
+
+/datum/eldritch_knowledge/curse/on_finished_recipe(mob/living/user,list/atoms,loc)
+
+	var/list/compiled_list = list()
+
+	for(var/mob/living/carbon/human/H in GLOB.human_list)
+		if(fingerprints[md5(H.dna.uni_identity)])
+			compiled_list |= H.real_name
+			compiled_list[H.real_name] = H
+
+	if(compiled_list.len == 0)
+		to_chat(user, "<span class='warning'>The items don't posses required fingerprints.</span>")
+		return
+
+	var/chosen_mob = input("Select the person you wish to curse","Your target") as null|anything in sortList(compiled_list, /proc/cmp_mob_realname_dsc)
+	if(!chosen_mob)
+		return ..()
+	curse(compiled_list[chosen_mob])
+	addtimer(CALLBACK(src, .proc/uncurse, compiled_list[chosen_mob]),timer)
+	return ..()
+
+/datum/eldritch_knowledge/curse/proc/curse(mob/living/chosen_mob)
+	return
+
+/datum/eldritch_knowledge/curse/proc/uncurse(mob/living/chosen_mob)
+	return
+
+/datum/eldritch_knowledge/summon
+	var/mob/living/mob_to_summon
+
+
+/datum/eldritch_knowledge/summon/on_finished_recipe(mob/living/user,list/atoms,loc)
+	message_admins("[initial(mob_to_summon.name)] is being summoned by [user.real_name] in [loc]")
+	var/list/mob/dead/observer/candidates = pollCandidatesForMob("Do you want to play as a [initial(mob_to_summon.real_name)]", ROLE_HERETIC, null, ROLE_HERETIC, 50,initial(mob_to_summon))
+	if(!LAZYLEN(candidates))
+		to_chat(user,"<span class='warning'>No ghost could be found...</span>")
+		return
+	var/mob/living/summoned = new mob_to_summon(loc)
+	var/mob/dead/observer/C = pick(candidates)
+	message_admins("[key_name_admin(C)] has taken control of ([key_name_admin(summoned)]).")
+	summoned.ghostize(0)
+	summoned.key = C.key
+
+	to_chat(summoned,"<span class='warning'>You are bound to [user.real_name]'s' will! Don't let your master die, protect him at all cost!</span>")
+///////////////
+///Base lore///
+///////////////
+
+/datum/eldritch_knowledge/spell/basic
+	name = "Break of dawn"
+	desc = "Starts your journey in the mansus. Allows you to get a select a target using a living heart on a transmutation rune."
```

Comment:
'Allows you to get a select a target'

Seems like you rewrote this part and forgot where you started.

Reply:
yeah